content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -314,6 +314,33 @@ on:
     types: [created, deleted]
 ```
 
+The `issue_comment` event occurs for comments on both issues and pull requests. To determine whether the `issue_comment` event was triggered from an issue or pull request, you can check the event payload for the `issue.pull_request` property and use it as a condition to skip a job.
+
+For example, you can choose to run the `pr_commented` job when comments are added to a pull request, and the `issue_commented` job when comments are added to an issue.
```

Comment:
@Simran-B sorry for the back and forth! Based on your observation in https://github.com/github/docs/pull/438/files#r517500086, I think we should align this comment too!

```suggestion
For example, you can choose to run the `pr_commented` job when comments events occur in a pull request, and the `issue_commented` job when comments events occur in an issue.
```

Reply:
@rachmari Good catch! And no worries, I appreciate your feedback very much.

```suggestion
For example, you can choose to run the `pr_commented` job when comment events occur in a pull request, and the `issue_commented` job when comment events occur in an issue.
```